### PR TITLE
Feature: Add Program class

### PR DESCRIPTION
**Pull request by licinamaja**
- Added new class FestivalProgram, since class name Program is already taken
- Added methods for adding movies and geting data

### CODE REVIEW

**Review by marsicdev — DISMISSED** [review]
LGTM

**Review by petardos — CHANGES_REQUESTED** [review]
add bracket at the end of GetData method

File: MovieFestival/FestivalProgram.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿namespace MovieFestival
+{
+	public class FestivalProgram
```
Comment (Mihailo-Pacaric) [inline MovieFestival/FestivalProgram.cs:3]: Program is available now as we changed the main class to App

**Review by marsicdev — DISMISSED** [review]

**Review by Mihailo-Pacaric — CHANGES_REQUESTED** [review]
Class name should be changed so we have consistency later

**Review by marsicdev — APPROVED** [review]
👍🏻

**Review by Mihailo-Pacaric — APPROVED** [review]
LGTM

**Review by petardos — APPROVED** [review]
LGTM

**Pull request merged**